setup.py

```diff
@@ -38,7 +38,7 @@ def get_version(filename):
     zip_safe=False,
     include_package_data=True,
     platforms='any',
-    install_requires=['pytest>=2.9', 'termcolor>=1.1.0'],
+    install_requires=['pytest>=2.9', 'termcolor>=1.1.0', 'packaging>=14.1'],
```

Comment:
FYI instead of adding a dependency, I've had good experiences (for years now) with:
```
from distutils.version import LooseVersion
if LooseVersion(a) <= LooseVersion(b)
```

Reply:
As does NumPy, SciPy, etc.

Reply:
Yeah, I was just interested on why it is considered to be better.
/cc @RonnyPfannschmidt

Reply:
It is the official tooling made by the [PyPA](https://www.pypa.io/en/latest/)